internal/dockerutil/image.go

```diff
@@ -75,6 +75,11 @@ func NewImage(logger *zap.Logger, cli *client.Client, networkID string, testName
 	return i
 }
 
+const (
+	LogTailAll     = 0
```

Comment:
Curious, can we instead default to all logs all the time?

Then we wouldn't need to worry about this extra argument and complexity.

Reply:
After digging a bit, for the `Exec` case, all logs are probably fine. The tail was more useful for the daemons to log out the end of their logs in case of a test failure for example. I will change this to not tail for the `Exec` jobs.

We lost the daemon logs which were printed out with tail 50 in the case of an env var in this PR, it would be nice to bring them back in some form:
https://github.com/strangelove-ventures/ibctest/pull/173/files#diff-4a9052564209349d7ee9d73566c8e6a5ef2e18983757bbdfcabc2912c38b4bb0

Reply:
I brought back the daemon tailing in case of test failure and added `SHOW_CONTAINER_LOGS` and `CONTAINER_LOG_TAIL` env vars for use when debugging.